webui/react/src/hooks/useTelemetry.ts

```diff
@@ -82,7 +82,7 @@ class Telemetry {
           clusterName: info.clusterName,
           edition: 'OSS',
           masterId: info.masterId,
-          username: auth.user.username,
+          userId: auth.user.id,
```

Comment:
**chore:** for segment traits, the user ID trait key is `id` according to [this](https://segment.com/docs/connections/spec/identify/#traits)

Reply:
reverting changes to this file